pkg/config/config.go

```diff
@@ -104,20 +105,30 @@ func FilePath(v *viper.Viper) (string, error) {
 		return p, nil
 	}
 
+	home := WakaHomeDir()
+	if home == "" {
+		return "", errors.New("failed getting user's home directory")
+	}
+
+	return filepath.Join(home, defaultFile), nil
+}
+
+// WakaHomeDir returns the current user's home directory.
+func WakaHomeDir() string {
```

Comment:
In Go we check for `err != nil` in most scenarios. In this case we'd refactor to something like this, what do you think?

```Go
unc WakaHomeDir() string {
	home, exists := os.LookupEnv("WAKATIME_HOME")
	if exists && home != "" {
		home, err := homedir.Expand(home)
		if err != nil {
			goto fallback
		}

		return home
	}

fallback:
	home, err := os.UserHomeDir()
	if err != nil {
		return pwd.Getpwuid(uint32(os.Getuid())).Dir
	}

	return home
}
```